### DISCUSSION TITLE
Python logging displaying in console

### DISCUSSION THREAD

**Discussion by ChadGoymer** [posit-dev/positron#4169] [Q&A]
Is anyone having trouble displaying Python logging in the console? The code below should print messages (for warnings and above as that is the default level for the logging module). However, I do not see any output:

![image](https://github.com/user-attachments/assets/cc20f3ba-6c51-4a66-b6a6-adb474b33042)

Contrast this with the same code, run on the same Python interpreter, in VS Code:

![image](https://github.com/user-attachments/assets/ce2b0eb7-beca-4bc7-ae4b-186e2b2a0795)

Is there something I am missing in Positron?

**Comment by seeM** [comment]
Thank you for reporting this! It looks like a bug, so I've logged an issue: https://github.com/posit-dev/positron/issues/4182.

**Reply by ChadGoymer** [reply]
Thanks for the quick response. Positron is a great IDE, I'm looking forward to the first official release.
